pos_minimize_menu/static/src/js/ProductScreen.js

```diff
@@ -0,0 +1,35 @@
+/*
+    Copyright (C) 2023 - Today: GRAP (http://www.grap.coop)
+    @author: Sylvain LE GAL (https://twitter.com/legalsylvain)
+*/
+odoo.define("pos_minimize_menu.ProductScreen", function (require) {
+    "use strict";
+
+    const ProductScreen = require("point_of_sale.ProductScreen");
+    const Registries = require("point_of_sale.Registries");
+
+    const OverloadProductScreen = (OriginalProductScreen) =>
+        class extends OriginalProductScreen {
+            get importantControlButtons() {
+                var importantButtons = [];
+                console.log("=============================");
```

Comment:
Is it a good practice to leave the console.log ?

Reply:
hehe ! absolutely not ! thanks ! 
Fixed.